notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>It's okay to link to the other notebook in the introduction, but not throughout your own notebook. Adding links to documentation would be better. <a href="https://mast.stsci.edu/spectra/docs/" target="_blank">https://mast.stsci.edu/spectra/docs/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>